src/components/ToolTip.vue

```diff
@@ -9,6 +9,7 @@ const warningIcon = require('src/assets/icon--warning.svg');
 const props = defineProps<{
     text?: string,
```

Comment:
required

Reply:
Doesn't need to be required, it is intentionally optional. You can have a warning without also having non warning text